application/controllers/Campaign.php

```diff
@@ -561,6 +561,8 @@ public function status($id = null, $component = null, $selected_milestone = null
             $this->db->where('cfo_act_agency', 'true');
         } else if ($id == 'omb-monitored') {
             $this->db->where('omb_monitored', 'true');
+        } else if ($id == 'long_running') {
```

Comment:
I am not responsible for the convention, but for whatever reason, the command-line option should use dashes to be consistent with the others, even though the column name uses underscores.